content/client-server-api/modules/end_to_end_encryption.md

```diff
@@ -720,35 +744,17 @@ devices:
           |                                 |
 ```
 
-###### Error and exception handling
+###### SAS error handling
 
 At any point the interactive verification can go wrong. The following
 describes what to do when an error happens:
```

Comment:
We should make it clear that these are only the failure modes specific to SAS verification, and link to the general section.